### PR TITLE
networkd: add support for bridge fdb destination address.

### PR DESCRIPTION
**Pull request by ssahani**
example conf:
```
[Match]
Name=vxlan1309

[BridgeFDB]
MACAddress=00:00:00:00:00:00
DestinationAddress=10.0.0.2

[BridgeFDB]
MACAddress=00:00:00:00:00:00
DestinationAddress=10.0.0.4

[BridgeFDB]
MACAddress=00:00:00:00:00:00
DestinationAddress=10.0.0.5

```
closes https://github.com/systemd/systemd/issues/5145

### CODE REVIEW

File: src/network/networkd-fdb.c [hunk 1]
```diff
@@ -258,3 +265,41 @@ int config_parse_fdb_vlan_id(
...
+
+        r = fdb_entry_new_static(network, filename, section_line, &fdb_entry);
+        if (r < 0)
+                return log_oom();
+
+        r = in_addr_from_string_auto(rvalue, &fdb_entry->family, &fdb_entry->destination_addr);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r,
+                           "FDB destination IP address is invalid, ignoring assignment: %s", rvalue);
+                return r;
```
Comment (yuwata) [inline src/network/networkd-fdb.c]: `return log_syntax()`

File: src/network/networkd-fdb.c [hunk 2]
```diff
@@ -145,6 +145,13 @@ int fdb_entry_configure(Link *link, FdbEntry *fdb_entry) {
                         return rtnl_log_create_error(r);
         }
 
+
+        if (!in_addr_is_null(fdb_entry->family, &fdb_entry->destination_addr )) {
```
Comment (yuwata) [inline src/network/networkd-fdb.c]: Unnecessary white space after `destination_addr`.

**Comment by ssahani** [comment]
updated thanks for review.

**Comment by yuwata** [comment]
@ssahani I slightly revised your commit, and on that add a simple test for the setting. PTAL.

**Comment by ssahani** [comment]
LGTM thanks

**Pull request merged**